recipes/bioconductor-fraser/meta.yaml

```diff
@@ -1,4 +1,4 @@
-{% set version = "1.99.4" %}
+{% set version = "1.99.3" %}
```

Comment:
This looks like a downgrade. Please explain

Reply:
@martin-g This downgrade is needed for a specific version of [drop](https://github.com/bioconda/bioconda-recipes/pull/48083) to ensure reproducibility. 
FRASER 1.99.4 has [some changes](https://github.com/gagneurlab/FRASER/commit/e775832aef77c14aa3c414832735bd1b0b9a4e47) that makes it incompatible with newer versions of drop. In drop v1.3.4, we will fix the version of FRASER to 1.99.3, just before those changes are reflected in FRASER 1.99.4.  
We will fix the bioconductor-fraser version to 1.99.3 same way as bioconductor-outrider version in drop [yaml file](https://github.com/bioconda/bioconda-recipes/pull/48083/files) 
There will be a new drop version soon after this, that will work with FRASER >=1.99.4

Reply:
OK!
In this case I think you need to create a sub-recipe for 1.99.3 instead of editing the main recipe and doing a downgrade in it.
Just create a folder named `1.99.3` next to the old/current `meta.yaml` and put the files from this PR in the new sub-folder.

Reply:
Thanks for the help! I changed the files as you explained.